src/main/java/Task.java

```diff
@@ -0,0 +1,38 @@
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    public String getDescription() {
+        return description;
+    }
+
+    public boolean getIsDone() {
+        return isDone;
+    }
+
+
+    // class initialization
```

Comment:
Please remove such comments as they go against the java coding standards.

Reply:
Okie I shall use java docs to do it Thank you!